requirements.txt

```diff
@@ -6,15 +6,15 @@ aiohttp-retry~=2.8
 click~=8.1
 nest-asyncio~=1.0
 ipython~=8.10
-nuclio-jupyter~=0.9.17
+nuclio-jupyter~=0.10.0
 numpy>=1.16.5, <1.27.0
 # pandas 2.2 requires sqlalchemy 2
 pandas>=1.2, <2.2
 # used as a the engine for parquet files by pandas
 # >=10 to resolve https://issues.apache.org/jira/browse/ARROW-16838 bug that is triggered by ingest (ML-3299)
 # <15 to prevent bugs due to major upgrading
 pyarrow>=10.0, <15
-pyyaml~=5.1
+pyyaml>=5.1, <7
```

Comment:
May you change it also here?
https://github.com/mlrun/mlrun/blob/500e7a2c5f38c66377eaae5181259c95c56e8df9/conda-arm64-requirements.txt#L5
Thanks in advance